tests/neg/i18750.check

```diff
@@ -0,0 +1,104 @@
+10 |class do; // error
+   |      ^^
+   |      an identifier expected, but 'do' found
+   |--------------------------------------------------------------------------------------------------------------------
+   | Explanation (enabled by `-explain`)
+   |- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   |
+   | If you want to use 'do' as identifier, you may put it in backticks: `do`.
+    --------------------------------------------------------------------------------------------------------------------
+-- [E040] Syntax Error: tests/neg/i18750.scala:11:7 --------------------------------------------------------------------
+11 |object if; // error // error
+   |       ^^
+   |       an identifier expected, but 'if' found
+   |--------------------------------------------------------------------------------------------------------------------
+   | Explanation (enabled by `-explain`)
+   |- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   |
+   | If you want to use 'if' as identifier, you may put it in backticks: `if`.
+    --------------------------------------------------------------------------------------------------------------------
+-- [E040] Syntax Error: tests/neg/i18750.scala:12:6 --------------------------------------------------------------------
+12 |trait else; // error
+   |      ^^^^
+   |      an identifier expected, but 'else' found
+   |--------------------------------------------------------------------------------------------------------------------
+   | Explanation (enabled by `-explain`)
+   |- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   |
+   | If you want to use 'else' as identifier, you may put it in backticks: `else`.
+    --------------------------------------------------------------------------------------------------------------------
+-- [E040] Syntax Error: tests/neg/i18750.scala:13:5 --------------------------------------------------------------------
+13 |type for; // error
+   |     ^^^
+   |     an identifier expected, but 'for' found
+   |--------------------------------------------------------------------------------------------------------------------
+   | Explanation (enabled by `-explain`)
+   |- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   |
+   | If you want to use 'for' as identifier, you may put it in backticks: `for`.
+    --------------------------------------------------------------------------------------------------------------------
+-- [E161] Naming Error: tests/neg/i18750.scala:11:0 --------------------------------------------------------------------
+11 |object if; // error // error
+   |^
+   |<error> clashes with class <error> in tests/neg/i18750.scala; the two must be defined together
```

Comment:
Note: This secondary error only on the `object` happens in combination with some of these other failures.

Reply:
are you working more on this?